trackData/frontend.json

```
@@ -74,7 +76,6 @@
         ],
         "examples": [
             "Defined and drove migration strategy to a new framework",
-            "Architected the new SSR with Vue.js on the wikidata team",
             "Understands all new frontend frameworks and chose the right one for the right task"
```

```suggestion
             "Understands the differences between frontend frameworks and their ecosystems and choses the right one for the right task"
```
What I mean by "ecosystems" is all the surrounding tools and plugins that frameworks come with, and also the community around the framework. That might not be the best wording.